lib/linux_admin/hosts.rb

```diff
@@ -36,6 +38,20 @@ def update_entry(address, hostname, comment = nil)
       end
     end
 
+    def hostname=(name)
```

Comment:
Do we want to add the logic to add the machine's IP address to the `/etc/hosts` file (https://github.com/ManageIQ/manageiq-appliance/blob/master/LINK/usr/bin/miqnet.sh#L167) here or have the caller do it in a separate call?
@Fryguy @bdunne